ui/raidboss/data/05-shb/raid/e9s.js

```diff
@@ -651,6 +752,72 @@ export default {
         },
       },
     },
+    {
+      id: 'E9S Summon',
+      netRegex: NetRegexes.ability({ id: '5019', source: 'Cloud Of Darkness', capture: false }),
+      netRegexDe: NetRegexes.ability({ id: '5019', source: 'Wolke Der Dunkelheit', capture: false }),
+      netRegexFr: NetRegexes.ability({ id: '5019', source: 'Nuage De Ténèbres', capture: false }),
+      netRegexJa: NetRegexes.ability({ id: '5019', source: '暗闇の雲', capture: false }),
+      netRegexCn: NetRegexes.ability({ id: '5019', source: '暗黑之云', capture: false }),
+      netRegexKo: NetRegexes.ability({ id: '5019', source: '어둠의 구름', capture: false }),
+      run: (data) => data.summon = true,
+    },
+    {
+      id: 'E9S Clone The Art Of Darkness',
+      netRegex: NetRegexes.startsUsing({ id: '561[EF]', source: 'Clone Of Darkness' }),
+      netRegexDe: NetRegexes.startsUsing({ id: '561[EF]', source: 'Klon der Dunkelheit' }),
+      netRegexFr: NetRegexes.startsUsing({ id: '561[EF]', source: 'Nuée de Ténèbres' }),
+      netRegexJa: NetRegexes.startsUsing({ id: '561[EF]', source: '幻影の雲' }),
+      netRegexCn: NetRegexes.startsUsing({ id: '561[EF]', source: '幻影之云' }),
+      suppressSeconds: 1,
+      promise: async (data) => {
+        const cloudOfDarknessLocaleNames = {
+          en: 'Cloud Of Darkness',
+          de: 'Wolke Der Dunkelheit',
+          fr: 'Nuage De Ténèbres',
+          ja: '暗闇の雲',
+          cn: '暗黑之云',
+          ko: '어둠의 구름',
+        };
+
+        const bossData = await window.callOverlayHandler({
+          call: 'getCombatants',
+          names: [cloudOfDarknessLocaleNames[data.parserLang]],
+        });
+
+        if (bossData === null || !bossData.combatants || !bossData.combatants.length)
+          return;
+
+        // All of the fake bosses have a BNpcID of 9020, 12379 is the real one.
+        const boss = bossData.combatants.filter((boss) => boss.BNpcID === 12379)[0];
+
+        data.boss = convertBossHeadingToClonePosition(boss);
+
+        const cloneOfDarknessLocaleNames = {
+          en: 'Clone Of Darkness',
+          de: 'Klon der Dunkelheit',
+          fr: 'Nuée de Ténèbres', // TODO: Does case sensitivity matter here? https://xivapi.com/BNpcName/9765 gives the name in lowercase
```

Comment:
Yes, `Nuée de Ténèbres` is spelled like this in the game. These are the capital letters of the punctuation.